charts/argocd-image-updater/README.md

```diff
@@ -14,15 +14,15 @@ helm repo add argo https://argoproj.github.io/argo-helm
 helm install argocd-image-updater argo/argocd-image-updater
 ```
 
-You will also need to run through the [secret setup documentation](https://argocd-image-updater.readthedocs.io/en/stable/install/start/#connect-using-argo-cd-api-server) so ArgoCD ImageUpdater can talk to the ArgoCD API (until its automated in this chart).
+You will also need to run through the [secret setup documentation] so ArgoCD ImageUpdater can talk to the ArgoCD API (until its automated in this chart).
```

Comment:
really nit-picky, but I believe it should be "Argo CD". Regardless, we should be consistent.

Reply:
@tico24 
Thank you for your review.
I fixed in https://github.com/argoproj/argo-helm/pull/2414/commits/923559370bb804ee1c79dc167deac80560591dd6 🙋